luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/DefaultLogicalTableGroupFactory.java

```diff
@@ -56,78 +70,94 @@ public LogicalTableGroup build(String name, ObjectNode configTable, LuthierIndus
         if (resourceFactories.getGranularityParser() == null) {
             throw new LuthierFactoryException(String.format(GRANULARITY_DICTIONARY_MISSING, name));
         }
-        LogicalTableGroup logicalTableGroup = new LogicalTableGroup();
```

Comment:
This is a pretty big method. I wonder if we might want to break it up into a sequence of helper methods,
`buildDimensions`, `buildMetricNames`, `buildPhysicalTables`, that sort of thing. Basically, anything that
requires any kind of looping gets pulled out into a helper method.

Reply:
done.